mypyc/analysis/dataflow.py

```diff
@@ -342,6 +358,9 @@ def visit_register_op(self, op: RegisterOp) -> GenAndKill:
     def visit_assign(self, op: Assign) -> GenAndKill:
         return set(), {op.dest}
 
+    def visit_set_mem(self, op: SetMem) -> GenAndKill:
+        return set(), {op.dest}
```

Comment:
We don't modify the pointer, so the the second return value item should be empty. We don't track memory locations in data flow analyses.